### ISSUE TITLE
Target Android version 9 missing + uses sdk build tools 27 instead of 28 (vs2019)

### ISSUE THREAD

**Issue by softlion** [xamarin/xamarin-android#2994]
### Steps to Reproduce

Create a default native android app using Xamarin.
Open android project properties on the manifest editor
Try to set Target Android version to 9. You can't.
Open AndroidManifest.xml using a text editor and add:
```
	<uses-sdk android:minSdkVersion="25" android:targetSdkVersion="28" />
```
Build
If you only have build tools v28+ installed, VS will stop building with an error "install build tools v27".

### Expected Behavior

Uses build tools v28

### Actual Behavior

VS stop building with an error:
```
Project 'Test.Droid' requires the following components installed on your machine: 
Android SDK Build-Tools 27.0.3
```

### Version Information

[version.txt](https://github.com/xamarin/xamarin-android/files/3088182/version.txt)

**Comment by dellis1972** [comment]
Looks like there is a disconnect between what the installer installers and what Xamarin android currently needs. 
We recently updated the `AndroidSdkBuildToolsVersion`, `AndroidSdkPlatformToolsVersion` to be version 28.x.x. But that is only on master at the moment, we are currently working with the sdkmanager team to resolve this issue.

To avoid this you can set 

```
<PropertyGroup>
   <AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">28.0.3</AndroidSdkBuildToolsVersion>
   <AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">28.0.0</AndroidSdkPlatformToolsVersion>
</PropertyGroup>
```

in your csproj and that should work around the issue.

**Comment by softlion** [comment]
Should i close the issue ?

**Comment by dellis1972** [comment]
@softlion no, we will use this to track the problem. We need to fix the sdkmanger and xamarin.android so they are instep with each other.

**Comment by softlion** [comment]
This is fixed in 16.1.0 preview 2.

**Comment by dellis1972** [comment]
@softlion thanks for confirming.
